packages/ios_platform_images/example/integration_test/ios_platform_images.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'package:ios_platform_images_example/main.dart' as app;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets(
+    'load ios bundled image',
+    (WidgetTester tester) async {
+      app.main();
+      await tester.pumpAndSettle();
+
+      expect(find.bySemanticsLabel('Flutter logo'), findsOneWidget);
```

Comment:
Are these tests actually doing anything? Isn't the *widget* there regardless of whether the image loads?

Reply:
You're right; should I check if the height and width are zero instead?